### PR TITLE
Feature - Update siteVersionPreviewPage to use ui-react

### PR DESCRIPTION
**Pull request by MoiyadKayda**
## Description
Positioning of container is still left.
We need a switch component in ui-react to fully remove material-ui dependency.

## Checklist:

- [x] I have updated the CHANGELOG with a summary of my changes

### CODE REVIEW

File: src/packages/console/src/pages/siteVersionPreviewPage.tsx [hunk 1]
```diff
@@ -214,26 +212,30 @@ export const SiteVersionPreviewPage = (props: ISiteVersionPreviewPageProps): Rea
...
                 {` ${siteVersion.name || 'Unnamed'}`}
-              </Typography>
-              {isEditable && <Typography color='textSecondary' className={classes.saveStatusText}>{savingError ? 'error saving!' : isSiteContentChanged || isSiteThemeChanged ? 'saving...' : 'saved'}</Typography>}
-              {!isEditable && <Typography color='textSecondary' className={classes.saveStatusText}>{'view-only mode'}</Typography>}
-              <Box className={classes.metaBoxSpacer} />
+              </Text>
+              {isEditable && <Text variant='light'>{savingError ? 'error saving!' : isSiteContentChanged || isSiteThemeChanged ? 'saving...' : 'saved'}</Text>}
+              {!isEditable && <Text variant='light'>{'view-only mode'}</Text>}
+              <Stack.Item growthFactor={1} shrinkFactor={1} />
+              {/* TODO: Need to create a switch component in ui-react */}
```
Comment (krishan711) [inline src/packages/console/src/pages/siteVersionPreviewPage.tsx]: use checkbox

File: src/packages/console/src/pages/siteVersionPreviewPage.tsx [hunk 2]
```diff
@@ -214,26 +212,30 @@ export const SiteVersionPreviewPage = (props: ISiteVersionPreviewPageProps): Rea
...
-          <div>Error loading site version. Please go back and try again...</div>
+          <Text>Error loading site version. Please go back and try again...</Text>
         ) : siteContent === undefined || siteTheme === undefined || assetFileMap === undefined ? (
-          <div>Loading...</div>
+          <Text>Loading...</Text>
         ) : (
           <React.Fragment>
-            <Box paddingX={2} paddingY={1} className={classes.metaBox}>
-              <Typography variant='subtitle1'>
+            <Stack direction={Direction.Horizontal} contentAlignment={Alignment.Start} childAlignment={Alignment.Center} shouldAddGutters={true} defaultGutter={PaddingSize.Wide} paddingLeft={PaddingSize.Wide} paddingVertical={PaddingSize.Default}>
```
Comment (krishan711) [inline src/packages/console/src/pages/siteVersionPreviewPage.tsx]: move stack to be directly under the containing view

File: src/packages/console/src/pages/siteVersionPreviewPage.tsx (hunk 1 above)
Comment (MoiyadKayda) [reply src/packages/console/src/pages/siteVersionPreviewPage.tsx]: Ok

File: src/packages/console/src/pages/siteVersionPreviewPage.tsx (hunk 2 above)
Comment (MoiyadKayda) [reply src/packages/console/src/pages/siteVersionPreviewPage.tsx]: image
Still the same problem
I think we need to add a custom margin or padding

Comment (krishan711) [reply src/packages/console/src/pages/siteVersionPreviewPage.tsx]: oh i wasnt trying to solve a problem, just general advice. This problem here is cos you shouldn't be using a containing view at all actually (it limits the width)

**Review by krishan711 — APPROVED** [review]

**Pull request merged**